Set up Buildbot using buildbot-nix in makemake (#228)
ngi-nix/ngipkgs: infra/docs/README.md

Reviewer: The file name can be `infra/README.md`

# NGI Infra Documentation

> ## Hosts / Inventory

Reviewer: ```suggestion
## Hosts
```

> We currently have a single host that's part of the NGI infrastructure. The host
> is called `makemake` and is a Hetzner physical node in Falkenstein, Germany.

Reviewer: ```suggestion
- `makemake`

  A Hetzner physical node in Falkenstein, Germany.
```

## Rebuilding `makemake`

> Under `/root/ngipkgs` a cloned version of this repository exists that is the
> source of truth for what is applied in `makemake`. In order to make changes to
> `makemake`, create a new branch with your changes in this repository, push it to
> github, pull your changes into `/root/ngipkgs` and switch to that branch. Then
> apply your changes with:
> 
> ```
> nixos-rebuild switch --show-trace -L --flake /root/ngipkgs/#makemake
> ```

Reviewer: ```suggestion
A clone of this repository is stored at `/root/ngipkgs`, and is the source of truth for what is applied in `makemake`. To make changes to `makemake`:
1. Create a new branch with your changes in this repository
1. Push it to GitHub
1. SSH into `makemake`:
  1. Pull your changes into `/root/ngipkgs` and switch to that branch.
  1. Apply your changes:

     ```
     nixos-rebuild switch --show-trace -L --flake /root/ngipkgs/#makemake
     ```
```
